.all-contributorsrc

```diff
@@ -1,6 +1,7 @@
 {
   "projectName": "react-testing-library",
   "projectOwner": "kentcdodds",
+  "repoType": "github",
```

Comment:
interesting, I ran `npm run add-contributor`, it put this in the wrong spot I guess?

Reply:
No worries. there was an update to `all-contributors` and I haven't updated my template yet :)